### PR TITLE
Bump cairo to v0.18.0

### PR DESCRIPTION
**Pull request by andrew-fleming**
This PR proposes to move the votes logic to common-components. This PR also proposes to include votes functionality for erc721.

I ran `yarn run update_scarb_project` on the test project and compiled...after 24 short minutes, there are no errors or warnings :)

### CODE REVIEW

File: packages/ui/src/cairo/ERC721Controls.svelte [hunk 1]
```diff
@@ -79,6 +79,37 @@
...
 
+<section class="controls-section">
+  <h1>
+    <!-- svelte-ignore a11y-label-has-associated-control -->
+    <label class="flex items-center tooltip-container pr-2">
+      <span>Votes</span>
+      <span class="ml-1">
+        <input type="checkbox" bind:checked={opts.votes}>
+      </span>
+      <HelpTooltip align="right" link="https://docs.openzeppelin.com/contracts-cairo/api/erc20#ERC20VotesComponent">
```
Comment (ericglau) [inline packages/ui/src/cairo/ERC721Controls.svelte:90]: This change should also be made in ERC20Controls.svelte

```suggestion
      <HelpTooltip align="right" link="https://docs.openzeppelin.com/contracts-cairo/api/governance#VotesComponent">
```

**Review by ericglau — APPROVED** [review]
LGTM in terms of the Wizard code, thanks!
I did not fully review the generated output though, it would be good if someone else can review those.

**Comment by ericnordelo** [comment]
Btw it took me 66 minutes to compile 😢

**Review by ericnordelo — APPROVED** [review]
LGTM!

**Review by immrsd — APPROVED** [review]
LGTM

**Comment by ericglau** [comment]
The `rollup` issue does not look correct, as I don't see version `2.79.2` being used anywhere in the project.

@SocketSecurity ignore npm/rollup@2.79.2

**Pull request merged**